server/tracer/src/main/java/org/apache/accumulo/tracer/TraceTableStats.java

```diff
@@ -33,14 +33,15 @@
 import org.apache.accumulo.core.security.Authorizations;
 import org.apache.accumulo.tracer.thrift.RemoteSpan;
 
+import com.beust.jcommander.Parameter;
+
 /**
  * Reads the trace table and prints out some stats about the spans found.
  */
 public class TraceTableStats {
-  static class Opts extends ClientOnDefaultTable {
-    public Opts() {
-      super("trace");
-    }
+  static class Opts extends ClientOpts {
+    @Parameter(names = "--table", description = "table to use")
+    String tableName = "trace";
```

Comment:
Why not just add tableName to ClientOpts?

Reply:
Options don't always require/use a table.

Reply:
Right but can't you set it to null like some of the options that aren't always used?

Reply:
I know you can set a parameter to default to null but that doesn't remove it as option. It'll still print out on the command line if user prints the help.

Reply:
Oh right, I forgot about the Help.